### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
Applied all html and css changes to recreate the desktop version of my portfolio project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,23 +10,246 @@
...
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>HTML</h4>
+              </div>
+            </div>
+            <div class="subcategorie" id="subcat3">
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>CSS</h4>
+              </div>
```
Comment (ishimwezachee) [inline index.html:204]: - [x] You should have space between your technologies, as it is in provided figma design, please make sure that you followed the design. 
image

- [ ] The background color should also be similar to the one from Figma design, the one you provided is too bright.

File: index.html [hunk 2]
```diff
@@ -10,23 +10,246 @@
 
 <body class="body">
```
Comment (ishimwezachee) [inline index.html:12]: - [x] In figma design the background color is not white, its is somehow gray, please use the right colors with the one provided from figma

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @juanmanuelbyc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
Git flow is implemented 👍 
Project is running successfully 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (juanmanuelbyc) [reply index.html:204]: done! thanks for feedback

File: index.html (hunk 2 above)
Comment (juanmanuelbyc) [reply index.html:12]: You're right. Thanks!

File: index.html [hunk 3]
```diff
@@ -10,23 +10,246 @@
...
+      <nav class="navbar">
+        <a class="my-logo" href="#">My Logo</a>
+        <div class="navlist">
+        <a class="navlink" href="#portfolio">Portfolio</a>
+        <a class="navlink" href="#about">About</a>
+        <a class="navlink" href="#contactme">Contact</a>
+        <a class="menu-icon" href="#"></a>
+        </div>
+      </nav>
+    </header>
```
Comment (codecaiine) [inline index.html:24]: - [ ] [OPTIONAL] Great Job with your desktop version :+1: , however, I think you could work on your navigation bar and make it stick as much as possible to the Figma template, it is very important as a full-stack developer to peer attention to details. please make sure to have the same colors, font, and background :smile:

### Figma template
Screenshot from 2022-07-07 09-59-46

### Your project
Screenshot from 2022-07-07 09-52-47

File: index.html [hunk 4]
```diff
@@ -10,23 +10,246 @@
...
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>HTML</h4>
+              </div>
+            </div>
+            <div class="subcategorie" id="subcat3">
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>CSS</h4>
+              </div>
```
Comment (codecaiine) [inline index.html:204]: - [ ] [OPTIONAL] Great work so far,  I would like to recommend you to use list `ul` and each technology has a `li` (e.g., "JavasScript", "HTML" and "CSS").

**Review by codecaiine — APPROVED** [review]
Hi @juanmanuelbyc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
-  Clean code :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**